be/src/Unic.UrlMapper2/code/Services/RedirectionService.cs

```diff
@@ -51,16 +51,16 @@ public virtual void PerformRedirect(RedirectSearchData redirectSearchData, HttpC
             var redirect = this.FilterRedirects(redirects, redirectSearchData);
             if (redirect is null) return;
 
-            var additionalTargetData = this.GetAdditionalTargetUrlData(redirectSearchData, redirect);
+            var additionalTargetData = this.GetAdditionalTargetUrlData(redirectSearchData.SourceTermOriginal, redirect);
 
             this.PerformRedirect(redirect, httpContext, additionalTargetData);
         }
 
-        protected virtual string GetAdditionalTargetUrlData(RedirectSearchData redirectSearchData, Redirect redirect)
+        protected virtual string GetAdditionalTargetUrlData(string sourceTerm, Redirect redirect)
```

Comment:
**issue:** This changes the method signature of a `protected virtual` method which is intended to be an extension point. This will break backwards compatibility.
**suggestion:** Revert this change and pass the full `RedirectSearchData` object.